lib/handlers/networkHandler.js

```diff
@@ -62,9 +62,11 @@ const responseHandler = response => {
 const getMatchingInterceptor = (interceptor, url) => {
   if (interceptor.count === 0) {
     return false;
+  } else if (url === interceptor.requestUrl) {
```

Comment:
@ed0906 sorry for the late response, but can we have a test for this in `networkHandler.test.js`?

Reply:
I was going to add one but then I saw how few tests were there already for this class

Reply:
It's ok to just add tests around this use case for the PR.